Css position edit (#1456)
Codecademy/docs: content/css/concepts/position/terms/position/position.md

  - 'paths/full-stack-engineer-career-path'
---

> The __position__ property defines the location of an element.

Reviewer: ```suggestion
The **position** property defines how elements are placed within the document.
```

I think "placed" makes more sense

Reviewer: Can we also edit the `Description` on line # 3 to say the following?

> Defines how elements are placed within the document.

## Syntax

> 

Reviewer: Below, on line # 20, let's change from \```css to \```pseudo.

- `fixed`: The element is removed from the document flow just like the absolute positioned elements. However, unlike the absolute positioned elements, which are relative to an ancestor element, the fixed elements are always relative to the document.
- `sticky`: This value is a mix between the relative and fixed positions. The element is treated as relative until a specific point or threshold is met, at which point the element is then treated as fixed.

## Example 1

> Set the position of multiple `.box` elements with different `position` values. The first box is sticky to the top of the document. The second is set to a fixed position 40px from the top of the page. The third box position is 80px from the element's top edge.

Reviewer: ```suggestion
The following example features three `.box` elements with different `position` values:
```

I would place the latter sentences after the image you provided.

```css
.box1 {
  height: 100px;
  width: 100px;
  background-color: red;
  position: sticky;
  top: 30px;
}

.box2 {
  height: 100px;
  width: 100px;
  background-color: blue;
  position: fixed;
  top: 80px;
}

.box3 {
  height: 100px;
  width: 100px;
  background-color: green;
  position: relative;
  top: 80px;
}
```

> Here's what the above examples look like:
> ![CSS Position Example](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-position-example.png "CSS position examples")

Reviewer: ```suggestion
Here's what the above examples look like:

![CSS Position Example](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-position-example.png)

The first box is sticky to the top of the document. The second is set to a fixed position 40px from the top of the page. The third box position is 80px from the element's top edge.
```
